src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,869 @@
+
+This function is called with a tuple of three elements:
+
+- `target`: [0x026b2f158255beac46c1e7c6b8bbf29a4b6a7b76](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/validation/standard/standard-versions-mainnet.toml#L21) - Mainnet OPContractsManager v2.0.0
+- `allowFailure`: false
+- `callData`: `0xff2dd5a1...` (output from the previous section)
+
+Command to encode:
+```bash
+cast calldata 'aggregate3((address,bool,bytes)[])' "[(0x026b2f158255beac46c1e7c6b8bbf29a4b6a7b76,false,0xff2dd5a100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000003000000000000000000000000229047fed2591dbec1ef1118d64f7af3db9eb290000000000000000000000000543ba4aadbab8f9025686bd03993043599c6fb04039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d90000000000000000000000007a8ed66b319911a0f3e7288bddab30d9c0c875c300000000000000000000000089889b569c3a505f3640ee1bd0ac1d557f436d2a039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d900000000000000000000000062c0a111929fa32cec2f76adba54c16afb6e8364000000000000000000000000d56045e68956fce2576e680c95a4750cf8241f79039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9)]"
+```
+
+The resulting calldata sent from the ProxyAdminOwner safe is thus:
+
+```
+0x82ad56cb000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000026b2f158255beac46c1e7c6b8bbf29a4b6a7b76000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000164ff2dd5a100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000003000000000000000000000000229047fed2591dbec1ef1118d64f7af3db9eb290000000000000000000000000543ba4aadbab8f9025686bd03993043599c6fb04039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d90000000000000000000000007a8ed66b319911a0f3e7288bddab30d9c0c875c300000000000000000000000089889b569c3a505f3640ee1bd0ac1d557f436d2a039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d900000000000000000000000062c0a111929fa32cec2f76adba54c16afb6e8364000000000000000000000000d56045e68956fce2576e680c95a4750cf8241f79039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d900000000000000000000000000000000000000000000000000000000
+```
+
+In mainnet runbooks, this calldata should appear in the [Action Plan](https://gov.optimism.io/t/upgrade-proposal-13-opcm-and-incident-response-improvements/9739#p-43725-action-plan-15) section of the Governance proposal.
+
+## Tenderly State Changes
+[Link](https://dashboard.tenderly.co/oplabs/eth-mainnet/simulator/ab8d86e3-d143-4e0b-9686-3dcab85a8609)
+
+## Auto Generated Task State Changes
```

Comment:
Looks like you've got some hold headings etc in this file, can you make it so it's following the same structure as this file? https://github.com/ethereum-optimism/superchain-ops/pull/784/files#diff-1eff5a492090ee81aa05524b6a8380071884cc0c8e7b57ab0590b9bb10bb22ff

Reply:
2f02882a4bf4